system-x/services/azure/key-vault/src/main/java/software/tnb/azure/keyvault/service/KeyVault.java

```diff
@@ -0,0 +1,22 @@
+package software.tnb.azure.keyvault.service;
+
+import software.tnb.azure.keyvault.account.KeyVaultAccount;
+import software.tnb.azure.keyvault.client.AzureKeyVaultClient;
+import software.tnb.azure.keyvault.validation.KeyVaultValidation;
+import software.tnb.common.service.Service;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+public class KeyVault extends Service<KeyVaultAccount, AzureKeyVaultClient, KeyVaultValidation> {
+
+    @Override
+    public void afterAll(ExtensionContext context) throws Exception {
+    }
+
+    @Override
+    public void beforeAll(ExtensionContext context) throws Exception {
+        account = new KeyVaultAccount();
+        client = new AzureKeyVaultClient();
+        validation = new KeyVaultValidation(client(), account());
```

Comment:
only validation should be here, account is automatically created when calling `account()` method, the same with client

Reply:
Changed